packages/app-store/ee/routing-forms/jsonLogicToPrisma.ts

```diff
@@ -0,0 +1,25 @@
+export const jsonLogicToPrisma = (jsonLogic) => {
+  if (!jsonLogic) {
+    return {};
+  }
+  jsonLogic = jsonLogic.logic;
```

Comment:
Can we store this in a different variable, as the parameter is directly getting manipulated